### DISCUSSION TITLE
intersection Observer Api calling frequently when it saw the element for the first time why ?

### DISCUSSION THREAD

**Discussion by GD-0606** [thebuilder/react-intersection-observer#665] [Q&A]
here, I am trying to apply the animation whenever the element comes into view w.r.t viewport .
but, I am facing an issue whenever the scroll position reaches my element then the animation tries to apply frequently
why this is happening can anyone help me out with this?

**Comment by thebuilder** [comment]
Most likely your element is changing size, causing it to trigger again.

**Reply by GD-0606** [reply]
can you give me suggestions to overcome this issue?

**Reply by thebuilder** [reply]
Not without seeing some code 😉
